specification/columns/commitmentdiscountunit.md

```diff
@@ -7,8 +7,8 @@ The CommitmentDiscountUnit column adheres to the following requirements:
 * CommitmentDiscountUnit MUST be present in a FOCUS dataset when the provider supports [*commitment discounts*](#glossary:commitment-discount).
 * CommitmentDiscountUnit MUST be of type String, and the units of measure used in CommitmentDiscountUnit SHOULD adhere to the values and format requirements specified in the [UnitFormat](#unitformat) attribute.
 * The CommitmentDiscountUnit MUST be the same across all *rows* where *CommitmentDiscountQuantity* has the same [*CommitmentDiscountId*](#commitmentdiscountid).
-* CommitmentDiscountUnit MUST NOT be null when *CommitmentDiscountId* is not null.
-* CommitmentDiscountUnit MUST be null in all other cases.
+* CommitmentDiscountUnit MUST NOT be null when *CommitmentDiscountId* is not null, and [*ChargeClass*](#chargeclass) is not "Correction".
+* CommitmentDiscountUnit MUST be null in all other cases when *CommitmentDiscountId* is not null.
```

Comment:
I believe this is a bug. This [table](https://docs.google.com/spreadsheets/d/1SvaD-PSzxwRKExKYbQXbtEmVPtbhN4HnycOatvLfGPI/edit?gid=1678090383#gid=1678090383) should help with the verification.

```suggestion
* CommitmentDiscountUnit MUST be null in all other cases.
```